src/components/Subscribe/_subscribe.scss

```diff
@@ -4,34 +4,57 @@
 @import '#scss/typography';
 
 .#{$px}-subscribe {
-  max-width: 30rem;
+  max-width: 600px;
   width: 100%;
 
+  .#{$px}-input {
+    &__label {
+      color: $pure-black;
+      font-size: 0.75rem;
+      text-transform: none;
+    }
+
+    &__validation {
+      padding-left: 0;
+    }
+  }
+
   &__title {
     @include text($heading3);
 
     margin-bottom: $spacing-xsm;
-    text-align: center;
   }
 
   &__blurb {
     @include text($body2);
 
-    text-align: center;
-  }
-
-  @include media($size-sm) {
-    &__title,
-    &__blurb {
-      text-align: left;
-    }
+    color: $pure-black;
   }
 
   &__input__wrapper {
     margin-top: $spacing-md;
   }
 
   &__button {
-    margin: $spacing-md 0 $spacing-md;
+    height: 36px;
+    margin: $spacing-sm 0 $spacing-md;
+    width: 100%;
+
+    @include media($size-sm) {
+      width: 240px;
+    }
+  }
+
+  &__privacy {
+    @include text($body3);
+
+    // margin-top: $spacing-sm;
+    background-color: yellow;
```

Comment:
Where in the figma are you seeing this background color applied. I can not find it.